src/components/Table/utils.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
I think a more appropriate name would be `columns.js` / `common.js`, etc... because these aren't really utils, more like helper components